package.json

```diff
@@ -100,15 +100,15 @@
     "dom-serializer": "^0.2.2",
     "domhandler": "^4.3.0",
     "expensify-common": "git+ssh://git@github.com/Expensify/expensify-common.git#398bf7c6a6d37f229a41d92bd7a4324c0fd32849",
-    "expo": "^49.0.0",
-    "expo-image": "1.8.1",
+    "expo": "^50.0.0-preview.7",
```

Comment:
Is this essential for RN 0.73.1? I don't like bumping to beta version like this.

Reply:
Yeah, each `expo` SDK version is connected to a specific RN version so we cannot decouple it 😅